plugins/github-pull-requests-board/src/components/EntityTeamPullRequestsCard/EntityTeamPullRequestsCard.tsx

```diff
@@ -28,11 +28,21 @@ import { PRCardFormating } from '../../utils/types';
 import { DraftPrIcon } from '../icons/DraftPr';
 import { useUserRepositories } from '../../hooks/useUserRepositories';
 
-const EntityTeamPullRequestsCard: FunctionComponent = () => {
+/** @public */
+export interface EntityTeamPullRequestsCardProps {
+  defaultLimit?: number;
+}
+
+const EntityTeamPullRequestsCard: FunctionComponent<
```

Comment:
I think we should be able to drop the FunctionComponent entirely in both places where it's used. I'm not really sure why it was there in the first place.

I believe this should work just as well and simplify the code.
```tsx
const EntityTeamPullRequestsCard = (props: EntityTeamPullRequestsCardProps) => {`